Assignment1/RegExpr.cs

```diff
@@ -1,10 +1,65 @@
 namespace Assignment1;
 
+using System.Text.RegularExpressions;
+
 public static class RegExpr
 {
-    public static IEnumerable<string> SplitLine(IEnumerable<string> lines) => throw new NotImplementedException();
+    public static IEnumerable<string> SplitLine(IEnumerable<string> lines)
+    {
+        var splitPattern = @"[^\w\d]";
```

Comment:
Pro tip: I stedet for ^\w\d, kan du skrive \W\D. Capitalized er det samme som NOT :)
Man kan også nøjes med at sige [\w\d]* i stedet. De her eksempler gør det præcis samme, det er bare lidt mere effektive måder at skrive det på :)